lib/build/build.go

```diff
@@ -46,6 +47,8 @@ var (
 	}
 )
 
+const versionExtraAllowedChars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-. "
```

Comment:
I'm aware this is not particularly inclusive or culturally sensitive, however I think for the version string it's fine to limit ourselves to basic ASCII.